memory-bank/04-development-workflow.md

```diff
@@ -0,0 +1,42 @@
+# Apache Traffic Server - Development Workflow
+
+## Development Workflow
+
+### Code Standards
+- **C++20** standard compliance
+- **clang-format** for code formatting
+- **yapf** for Python code formatting
+- **cmake-format** for CMake files
+
+### Testing
+- **Regression tests**: `BUILD_REGRESSION_TESTING=ON`
+- **AuTest**: Automated testing framework
+- **Fuzzing**: Clang fuzzer support
+- **CI/CD**: Jenkins-based continuous integration
+
+### Contributing Process
+1. Fork and branch from `master`
```

Comment:
Should have the commands, would be cool to have one of those boxes of commands you can copy from.